packages/create-fuels/src/lib/getPackageManager.ts

```diff
@@ -25,24 +21,22 @@ export const packageMangers = {
   },
 } as const;
 
-export const getPackageManager = (opts: ProgramOptions) => {
-  const packageMangerOpts = {
-    pnpm: opts.pnpm,
-    npm: opts.npm,
-    bun: opts.bun,
-  };
-
-  const cliChosenPackageManagerSelected = Object.entries(packageMangerOpts)
-    .filter(([, v]) => v)
-    .map(([k]) => k) as PackageManager[];
+export function getUserPkgManager(): PackageManager {
+  const userAgent = process.env.npm_config_user_agent || '';
 
-  let packageManager: PackageManager | undefined = cliChosenPackageManagerSelected[0];
-  if (cliChosenPackageManagerSelected.length > 1) {
-    warn('More than one package manager was selected.');
+  if (userAgent.startsWith('pnpm')) {
+    return 'pnpm';
   }
 
-  if (!packageManager) {
-    packageManager = 'npm'; // default to npm if the user has not specified a package manager (eg. --pnpm, --bun)
+  if (userAgent.startsWith('bun')) {
+    return 'bun';
   }
+
```

Comment:
`console.warn` here if the package users package manager is not supported and we default to `npm`?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3503/commits/6f3b3f9a93ea75e9af2926977917ec8f970be40d